react/fields/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component, PropTypes } from 'react';
+import styles from './Checkbox.less';
+import CheckMarkIcon from '../../icons/CheckMarkIcon/CheckMarkIcon';
+import classnames from 'classnames';
+
+function combineClassNames(props = {}, ...classNames) {
+  const { className, ...restProps } = props;
+
+  return {
+    className: classnames.apply(null, [...classNames, className]), // eslint-disable-line no-useless-call
+    ...restProps
+  };
+}
+
+export default class Checkbox extends Component {
+  static displayName = 'Checkbox';
+
+  static propTypes = {
+    id: PropTypes.string.isRequired,
+    label: PropTypes.string.isRequired,
+    className: PropTypes.string,
+    inputProps: PropTypes.shape({
+      onChange: PropTypes.func,
+      checked: PropTypes.bool
```

Comment:
True, I don't expect there to be a scenario where you would include a style guide component  and not have full control of its events and values.